front-end/src/components/StudentSubscriptions/StudentTable/StudentTable.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+// components
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { Button } from '@mui/material';
+import TableFooter from '@mui/material/TableFooter';
+import TablePagination from '@mui/material/TablePagination';
+import TablePaginationActions from '../TablePaginationActions/TablePaginationActions';
+
+// assets
+import avatar from '../../../assets/images/avatar.jpeg';
+
+// styles
+import styles from './StudentTable.module.scss';
+
+const StudentTable = ({ subscriptions, deleteSubscriptionById }) => {
```

Comment:
If this table showing for student, maybe will be better rename to Teacher table?
Or it could be a generic table for students and teachers subscriptions